services/wex/secure_token.md

```diff
@@ -0,0 +1,37 @@
+# Secure token
+
+When you start a new registration or renew an existing one a token is created that is then used to identify the in progress `transient_registration`. From a design point of view this delivers a number of benefits to the user (as long as they retain the link)
+
+- they are not tied to their current session. They can close the browser and return to the journey at a later date
+- we avoid needing to setup an account
+- we are not adding an additional cookie to their session to track the registration
+
+For example, once a user is past the first page of the new registration journey their browser will show something like this for the url
+
+> https://wasteexemptions.service.gov.uk/location/3zaSBUDkBQzhu8s5xd7Phoow
+
+## Security
+
+Because we're identifying the in-progress registration from a url parameter and not something stored in the cookie, we needed to make sure it's sufficiently complex to guess, and unique enough to avoid duplication.
```

Comment:
```suggestion
Because we're identifying the in-progress registration from a URL parameter and not something stored in the cookie, we needed to make sure it's sufficiently complex to guess, and unique enough to avoid duplication.
```